jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -59,9 +59,7 @@
 import org.gradle.api.java.archives.internal.DefaultManifest;
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.configuration.ConsoleOutput;
-import org.gradle.api.plugins.Convention;
-import org.gradle.api.plugins.JavaPluginConvention;
-import org.gradle.api.plugins.WarPluginConvention;
+import org.gradle.api.plugins.*;
```

Comment:
Formatting problem. Running `./build.sh format` at the repo root should auto-format all code.

Reply:
`./build.sh format` doesn't change anything.

Reply:
Hmm. Sorry, could you expand the import?

Reply:
I think you'll have to manually expand this then?